avr-asm.JSON-tmLanguage

```diff
@@ -2,106 +2,89 @@
     "name": "AVR ASM",
     "scopeName": "source.asm", 
     "fileTypes": ["asm", "inc"],
-    "patterns": [
-
+    "patterns": 
+    [
         {
-            "match": "(?i)\\br(0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31)\\b",
+            "match": "(?i)\\bR(0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31)\\b",
```

Comment:
Why did you make uppercase for general purpose registers? This does not take effect because regex is configured with ?i (case insensitive mode).